app/helpers/hosts_helper.rb

```diff
@@ -488,4 +489,23 @@ def randomize_mac_link
   def power_status_visible?
     SETTINGS[:unattended] && Setting[:host_power_status]
   end
+
+  def load_tabs(host)
+    @tabs = facet_tabs(host)
+  end
+
+  def host_tab(group, id, val, host_form)
+    content_tag(:div, :id => id, :class => "tab-pane") do
+      disable_tab(group) +
```

Comment:
We should talk about this. Is it safe to assume that every developer wants to use this functionality?